packages-content-model/roosterjs-content-model-editor/lib/publicApi/utils/paste.ts

```diff
@@ -67,7 +67,6 @@ export default function paste(
             } = triggerPluginEventAndCreatePasteFragment(
                 editor,
                 clipboardData,
-                null /* position */,
```

Comment:
@BryanValverdeU  want to check with you why this parameter is always passed as null, and it is ok I remove it?

Reply:
I think it can be removed. Just need to pass null to the inner util that is still used in the normal editor.

Reply:
Sounds good, thanks.